gui/src/i18n/fr/translation.json

```
@@ -4,15 +4,15 @@
     "connection-lost": "Connexion avec le serveur perdue. Reconnexion..."
   },
   "tips": {
-    "find-tracker": "Impossible de différencier vos capteurs? Bougez-les pour qu'ils brillent à l'écran.",
-    "do-not-move-heels": "Assurez-vous de ne pas bouger vos pieds pendant la calibration!"
+    "find-tracker": "Impossible de différencier vos capteurs ? Secouez-en un pour qu'il soit mis en avant.",
```

```suggestion
    "find-tracker": "Impossible de différencier vos capteurs ? Secouez-en un pour qu'il soit mis en évidence.",
```
I think this fits better?